packages/ui-components/tooltip/src/__snapshots__/withTooltip.test.tsx.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`withTooltip tests does not render a tooltip when not specified 1`] = `
+<body>
+  <div>
+    <div>
+      foobar
+    </div>
+  </div>
+</body>
+`;
+
+exports[`withTooltip tests renders a tooltip when specified 1`] = `
+<body>
+  <div>
+    <div>
+      foobar
+    </div>
+  </div>
+</body>
+`;
+
+exports[`withTooltip tests renders a tooltip when specified via tooltip props 1`] = `
+<body>
+  <div>
+    <div>
+      foobar
+    </div>
+  </div>
+</body>
+`;
+
+exports[`withTooltip tests renders a tooltip when specified without id 1`] = `
+<body>
+  <div>
+    <div>
+      foobar
+    </div>
+  </div>
+</body>
```

Comment:
tooltip supposed to be in these snapshots?

Reply:
Ah good point